From ee783c532efef4300360d0d84936650ab0091ef2 Mon Sep 17 00:00:00 2001
From: Peter Hutterer <peter.hutterer@who-t.net>
Date: Tue, 8 May 2012 13:55:17 +1000
Subject: [PATCH] lib: fix an error message

Leftover error message from when this was serial-only code.

Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
---
 libwacom/libwacom.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libwacom/libwacom.c b/libwacom/libwacom.c
index 821e3ee..359e945 100644
--- a/libwacom/libwacom.c
+++ b/libwacom/libwacom.c
@@ -229,7 +229,7 @@ get_device_info (const char   *path,
 
 		g_assert (product_str);
 		if (sscanf(product_str, "%d/%x/%x/%d", &garbage, vendor_id, product_id, &garbage) != 4) {
-			libwacom_error_set(error, WERROR_UNKNOWN_MODEL, "Unimplemented serial bus");
+			libwacom_error_set(error, WERROR_UNKNOWN_MODEL, "Unable to parse model identification");
 			g_object_unref(parent);
 			goto bail;
 		}
-- 
1.7.10.1

